coop_badge_reader/static/www/lib/odoo/dist/odoo.js

```diff
@@ -190,23 +189,38 @@ angular.module('odoo').provider('jsonRpc', function jsonRpcProvider() {
 			*/
 			function buildRequest(url, params) {
 				odooRpc.uniq_id_counter += 1;
-				if (odooRpc.shouldManageSessionId)
-					params.session_id = $cookies.session_id
+				
+				//if (params.model)
+				//    params.session_id = $cookies.session_id
+				//if (odooRpc.shouldManageSessionId)
+				//	params.session_id = $cookies.session_id
 
 				var json_data = {
 					jsonrpc: '2.0',
 					method: 'call',
 					params: params, //payload
 				};
-				return {
+				/*return {
 					'method' : 'POST',
 					'url' : odooRpc.odoo_server + url,
 					'data' : JSON.stringify(json_data),
 					'headers': {
 						'Content-Type' : 'application/json'
 					},
 					'id': ("r" + odooRpc.uniq_id_counter),
-				};
+				};*/
+				return {
+                    'method' : 'POST',
+                    'url' : odooRpc.odoo_server + url,
+                    'data' : JSON.stringify(json_data),
+                    'Content-Type' : 'application/json',
+                    'headers': {
+                        'X-Debug-Mode': '',
+                        'timeout' : 60000,
+                    },
+                    'id': ("r" + odooRpc.uniq_id_counter),
+                };
```

Comment:
Can you fix the indentation?
I think this file has mixed tabs and spaces indents